_posts/2019-03-12-so-you-want-to-be-an-engineer.md

```diff
@@ -0,0 +1,170 @@
+
+# Step One: Tell People What You Want
+
+This might've been the single biggest learning I took away from this experience: _if you tell people you want
+something, you might just get it._
+
+That may sound super obvious. It wasn't for me. I've usually been very passive in my career decisions, taking the
+path of least resistance and considering myself lucky when I was able to keep progressing. In this case, I was
+making a substantial departure from that idea by being proactive about what it was I wanted.
+[This post](https://engineering.gusto.com/i-didnt-want-to-be-a-manager-anymore-and-the-world-didnt-end/) by
+[Noa Elad](https://twitter.com/NoaElad) does a great job with this topic and is certainly worth a read.
+
+The first person I told at Artsy was [Orta](https://github.com/orta). He'd often encouraged me to develop my
+technical skills, and since he knows Artsy's engineering team and stack better than just about anyone, I figured
+he'd be able to point me in the right direction when it came to learning resources and navigating company politics
+to get to my eventual goal.
+
+The second person I told was my manager on the CRM team. I fortunately had a very good relationship with my manager
+and was confident that she would help me if she could. And by telling her early, I was giving her more opportunity
+to advocate for me and making it easier for her to replace me in the event that I was able to switch teams.
+
+The third person I told was Artsy's CTO, [dB](https://www.dblock.org/). This was Orta's recommendation—dB would be
+able to tell me if and when a move might be possible, and he could suggest things I should do to improve my chances
+of making the switch.
+
+I also didn't keep it a secret from the rest of my team or the company. I didn't show up wearing a shirt that said
+"ENGINEER" on it, but I told people, "I'm working on becoming an engineer. I'm really hoping to stay at Artsy, but
+if there's not a role open for me, that's fine—I'll search elsewhere."
+
+The net outcome of these conversations was that there wasn't a role open right then, but I also left with a better
+idea of what I should learn and what I could expect from the coming months.
+
+# Step Two: Figure Out What You Want to Learn
+
+The answer to that question really depends on who you are, where you work, and where you _want_ to work.
+
+I wanted to work at Artsy, and I felt that I was most interested in front-end work. So I asked a few of our
+engineers to help me understand our stack and to recommend frameworks/languages I should learn.
+
+If you're interested in changing companies as well as careers, it's worth seeing if your target company or
+companies have open source code you can check out. See if you can find them on [GitHub](https://www.github.com) and
+look at some of their recently updated apps. If you already have some coding experience, see if you can contribute
```

Comment:
> recently updated ~apps~ repos

Artsy's still fairly unusual (that we have OSS apps). Most companies releasing open source do tools, frameworks, etc.

Reply:
ah yep, that makes sense. updated!